### PR TITLE
Basic page layout

### PR DESCRIPTION
**Pull request by kerzhner**
Introduce a layout with a top bar and three content columns below.
Screenshot 2023-05-14 at 7 39 15 PM

### CODE REVIEW

File: packages/site/src/App.tsx [hunk 1]
```diff
@@ -25,24 +24,21 @@ function App() {
...
-          myBalanceFree={BigInt(50)}
-          theirBalanceFree={BigInt(200)}
-        ></NetworkBalance>
-
-        <p>The nitro client version is {version}</p>
-        <p>
-          The nitro node at {url} has address {address}
-        </p>
+      <TopBar url={url} setUrl={setUrl} />
+      <div style={{ display: "flex", justifyContent: "space-around" }}>
```
Comment (geoknee) [inline packages/site/src/App.tsx:28]: Are we doing inline css now, or should this be imported from a `.css` file?

File: packages/site/src/App.tsx [hunk 2]
```diff
@@ -25,24 +24,21 @@ function App() {
...
+        <div className="card">
+          <NetworkBalance
+            status="running"
+            lockedBalances={[]}
+            myBalanceFree={BigInt(50)}
+            theirBalanceFree={BigInt(200)}
```
Comment (geoknee) [inline packages/site/src/App.tsx:34]: Out of scope but I think there's an argument for ripping out any mocked data here. Given that we have storybook for mocking things out.

**Review by geoknee — APPROVED** [review]

File: packages/site/src/App.tsx (hunk 1 above)
Comment (kerzhner) [reply packages/site/src/App.tsx:28]: No strong preference. I would say let's start with css inline until the codebase is large enough to require a css file.

File: packages/site/src/App.tsx (hunk 2 above)
Comment (kerzhner) [reply packages/site/src/App.tsx:34]: Agreed. We should do that going forward.

**Pull request merged**